HTWDD/Main Categories/General/AppCoordinator.swift

```diff
@@ -115,13 +115,21 @@ extension AppCoordinator {
     func selectChild(coordinator: CoordinatorRoute?) {
         guard let coordinator = coordinator else { return }
         
-        // Dismiss any ViewController presented on tabBarController
-        if let presented = self.tabBarController.presentedViewController {
-            presented.dismiss(animated: false)
-        }
-        
         switch coordinator {
         case .schedule:
+            // Dismiss any ViewController presented on tabBarController
+            if let presented = self.tabBarController.presentedViewController {
```

Comment:
Is there a reason, why you put the `dismiss` in both cases? 
Or is it, in case we add a different route and it's not needed?

Reply:
Different routes might need a different behaviour, correct.